Sources/Playdate/Documentation.docc/BuildingTheExamples.md

```diff
@@ -93,3 +99,60 @@ Select the Run icon in the toolbar or press Cmd+R on your keyboard to start a bu
 
 > Note:
 > Learn more about [Xcode](https://developer.apple.com/xcode/)
+
+
+## Determining the toolchain
+
+When using `make` to build the examples, the latest installed Swift toolchain will be automatically determined. It may, however, sometimes be useful to specify a specific toolchain version. It is always necessary to define the toolchain when building with swift package manager in terminal.
+
+> Note: When using `make` to build the examples, Swift toolchains installed for the current user take precedence over globally installed toolchains.
+
+1. The toolchain identifier can be found under the `CFBundleIdentifier` key in the `Info.plist` file found at the root of the toolchain.
```

Comment:
Can we reference this content in DownloadingTheTools.md instead of duplicating it here?